src/content/api/_endpoints/bank-accounts-get-balance.js

```diff
@@ -1,14 +1,20 @@
 export default {
   method: 'GET',
-  path: '/api/bank-accounts/WRhAxxWpTKb5U7pXyxQjjY/balance',
+  path: '/api/accounts/Jaim1Cu1Q55uooxSens6yk/bank-account-balances',
   request: {
     headers: {
       'X-Api-Key': '<TOKEN>',
     },
   },
   response: {
-    bankAccountId: 'WRhAxxWpTKb5U7pXyxQjjY',
-    balance: 1000,
-    currency: 'NZD'
+    items: [
+      {
+        bankAccountId: 'b3f9ba1abff64775877d4a2fb073b123',
```

Comment:
@Wulingoh it looks like the bank account IDs in the path and this `bankAccountId` are mismatched.
Is this correct?

Reply:
@MeganSteenkamp, it is an accountId in the path. 😃